scripts/download_ossfuzz_corpus.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+set -e
+
+(
```

Comment:
Why is a subshell needed here?

Reply:
Because of the directory change. The other question would be: Why do we clone to the root directory? Perhaps we should at least check whether `$CI` is set? Or clone to `/tmp` instead?

Reply:
Thanks, removed subshell and moved corpus cloning to `/tmp`